grr/gui/api_client/client.py

```diff
@@ -56,6 +56,27 @@ def ListFlows(self, offset=0, count=0):
     return utils.MapItemsIterator(
         lambda data: flow.Flow(data=data, context=self._context),
         items)
+ 
+  def ValidApproval(self, offset=0, count=0):
```

Comment:
Let's rename it to "HasValidApproval" and remove "offset" and "count" arguments as they don't make sense in case of checking for valid approvals.